WalletWasabi.Gui/Controls/WalletExplorer/TransactionInfo.cs

```diff
@@ -56,5 +65,68 @@ public string TransactionId
 			get => _transactionId;
 			set => this.RaiseAndSetIfChanged(ref _transactionId, value);
 		}
+ 
+		public string TotalInputValue
+		{
+			get => _totalInputValue;
+			set => this.RaiseAndSetIfChanged(ref _totalInputValue, value);
+		}
+
+		public string TotalOutputValue
+		{
+			get => _totalOutputValue;
+			set => this.RaiseAndSetIfChanged(ref _totalOutputValue, value);
+		}
+		public int InputCount
+		{
+			get => _inputCount;
+			set => this.RaiseAndSetIfChanged(ref _inputCount, value);
+		}
+
+		public int OutputCount
+		{
+			get => _outputCount;
+			set => this.RaiseAndSetIfChanged(ref _outputCount, value);
+		}
+
+		public static TransactionInfo FromBuildTxnResult(BuildTransactionResult result)
+		{
+			var inputAddrMoney = result.Transaction.Transaction.Inputs.Select(x =>
+			{
+				if (result.Store.TransactionStore.TryGetTransaction(x.PrevOut.Hash, out var prevTxn))
+				{
+					var targetTxO = prevTxn.Transaction.Outputs[x.PrevOut.N];
+					var address = targetTxO.ScriptPubKey.GetDestinationAddress(result.Network).ToString();
+					var money = targetTxO.Value;
+
+					return new AddressMoneyTuple(address, money, false);
+				}
+				else
+				{
+					return AddressMoneyTuple.Empty;
+				}
+			}).Where(x => x != AddressMoneyTuple.Empty).ToList();
+
+			var outputAddrMoney = result.Transaction.Transaction.Outputs.Select(targetTxO =>
+			{
+				var address = targetTxO.ScriptPubKey.GetDestinationAddress(result.Network).ToString();
+				var money = targetTxO.Value;
+				return new AddressMoneyTuple(address, money, false);
+			}).ToList();
```

Comment:
There are parts that are repeated and the `AddressMoneyTuple.Empty` is needed for this code so, please consider making this as follow and remove the `AddressMoneyTuple.Empty`. Additionally, the `AddressMoneyTuple.ctor` could be simplified to accept `Script` and `Money` or, even better `TxOut`

```suggestion
			AddressMoneyTuple FromTxOutput(TxOut output) =>
				new AddressMoneyTuple(output.ScriptPubKey.GetDestinationAddress(result.Network).ToString(), output.Value);

			TxOut GetOutput(OutPoint outpoint) =>
				result.Store.TransactionStore.TryGetTransaction(outpoint.Hash, out var prevTxn)
					? prevTxn.Transaction.Outputs[outpoint.N]
					: null;

			var inputAddrMoney = result.Transaction.Transaction.Inputs
				.Select(x => x.PrevOut)
				.Select(GetOutput)
				.Where(x => x is { })
				.Select(FromTxOutput);

			var outputAddrMoney = result.Transaction.Transaction.Outputs.Select(FromTxOutput);

```